README.md

```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
```

Comment:
Please add here options to "To try Spectrum OS":
a. download binary image (link to arm-kal.us.to - let's change the link to vedenemo.dev hydra cross-compilation later)
b. "build it from source..."

Reply:
Done.